### ISSUE TITLE
A11y Fail: Time Picker - Helper text

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1381]
CENG_1966

https://react-magma.cengage.com/version/4.5.0/api/time-picker/#helper_message

### Issue 1:
The helper message that appears below the time picker appears after the input. Therefore users who cannot see the helper text may not have access to it until after that choose a time. We probably have this same issue in all other inputs using helper text.

### Suggestion:
Consider using `aria-describedby` to associate the first input with the helper text. In addition, the aria-live attribute should be removed from the content unless the content is dynamic. If the content is just static text that is always displayed then it should not be marked as a live region.

**Comment by silvalaura** [comment]
Issue 2 is just a problem with the Internationalization example. The code already supports using the i18n variable `amButtonAriaLabel` for the aria label.
